packages/flutter/lib/src/widgets/text.dart

```diff
@@ -209,6 +207,17 @@ class DefaultTextStyle extends InheritedTheme {
   }
 }
 
+class _NullWidget extends StatelessWidget {
+  const _NullWidget();
+
+  @override
+  Widget build(BuildContext context) {
+    throw FlutterError(
+      'DefaultTextStyle.fallback cannot be incorporated into the tree.'
```

Comment:
This doesn't really tell the user what to do about it.

Could you give more context? Like "...because no child widget was given" or something?

Reply:
```
  /// A const-constructable default text style that provides fallback values.
  ///
  /// Returned from [of] when the given [BuildContext] doesn't have an enclosing default text style.
  ///
  /// This constructor creates a [DefaultTextStyle] with an invalid [child], which
  /// means the constructed value cannot be incorporated into the tree.
  const DefaultTextStyle.fallback({ Key? key })
```

I don't know what to add. Do you have a suggestion?

Reply:
How about this:
```
A DefaultTextStyle constructed with DefaultTextStyle.fallback cannot be incorporated into the widget tree, it is meant only to provide a fallback value returned by DefaultTextStyle.of() when no enclosing default text style is present in a BuildContext.
```

Reply:
It's way better than my poor english would have suggested :) thanks!